ml-agents/mlagents/trainers/ppo/models.py

```diff
@@ -1,12 +1,115 @@
 import logging
 import numpy as np
+import sys
 
 import tensorflow as tf
 from mlagents.trainers.models import LearningModel
 
+# Variable scope in which created variables will be placed under
+TOWER_SCOPE_NAME = "tower"
+
 logger = logging.getLogger("mlagents.trainers")
 
 
+class PPOMultiGPUModel(object):
```

Comment:
This doesn't need to subclass from `object`, It can just be `class PPOMultiGPUModel:`